### PR TITLE
Fix Release Policy diagram typo

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Fixes an "Alpha/Slpha" typo in the Release Policy diagram:

Screenshot 2024-10-07 at 11 11 56

## When should this change go live?

- [X] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: versioned_docs/version-8.5/reference/img/diagram-releases.png
Comment (christinaausley) [inline versioned_docs/version-8.5/reference/img/diagram-releases.png:1]: Non-blocking, but I also noticed there is an extra space after the second "Minor (stable )" 😄 if we wanted to fix this in one go.

**Review by christinaausley — DISMISSED** [review]
One non-blocking comment 👍 Nice catch!

File: versioned_docs/version-8.5/reference/img/diagram-releases.png
Comment (mesellings) [reply versioned_docs/version-8.5/reference/img/diagram-releases.png:1]: Great spot! Yep, I'll change that - thank you for catching that!

**Comment by mesellings** [comment]
Thansk for the great catch - @christinaausley please re-review 👍

**Pull request merged**